BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-1.0.25-prerelease-00420-01
\ No newline at end of file
+1.0.25-prerelease-00424-01
```

Comment:
I notice CoreFx is now at 00425-03.  @mellinoe -- do you mind updating to match, or is there a reason not to?